diff --git a/index.js b/index.js
index 20c8e2ba5f5940a1d5fe00e1a4de4934c1a2229c..1f8f74355af9a62c79964b46d08e1580ffa96170 100755
--- a/index.js
+++ b/index.js
@@ -19,9 +19,8 @@ const run = async () => {
   if (ci && branch === 'master' && !process.env.INTERNAL_SKIP_CACHE) {
     await cache.save(results)
   }
-  const cachedResults = await cache.read()
 
-  const summary = summarize(results, cachedResults)
+  const summary = summarize(results, undefined)
   cli.report(summary)
 
   if (ci && flags.enableGitHubChecks) {